core/src/main/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoBigQuerySupport.scala

```diff
@@ -216,6 +218,13 @@ trait DataRepoBigQuerySupport {
 
       val schemaFields = queryResults.getSchema.getFields
 
+      // check results for data we don't handle well. For now this is just RECORD types.
+      if (schemaFields.asScala.exists(_.getType == LegacySQLTypeName.RECORD)) {
+        logger.warn(s"query results for entity type $entityType contains one or more fields " +
+          s"with ${LegacySQLTypeName.RECORD.toString} datatypes; these " +
+          s"are unsupported and will be serialized to a string value.")
+      }
+
```

Comment:
This log message lets us know we hit a `STRUCT`, but it doesn't log the snapshot id or the workspace .... we don't have access to that info without changing a bunch of method signatures. Is this worth it?